### PR TITLE
Dse/fix documentation

### PR DESCRIPTION
**Pull request by sellisd**
I moved the detailed documentation under the cosmotech-api chart and replaced the root readme with a more generic readme for the whole repository

### CODE REVIEW

**Review by jreynard-code — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,797 +1,29 @@
...
-
-### Step 1: Add Helm Repository
-
-If you haven't already added the Helm repository for Cosmo Tech API, execute the following command:
+## Installation
 
+Add the helm repository and update:
 ```bash
-helm repo add cosmotech https://cosmo-tech.github.io/helm-charts
+helm repo add cosmotech-api https://cosmo-tech.github.io/helm-charts/
```
Comment (vysahjk) [inline README.md:19]: Could you go back to cosmotech please ? cosmotech-api is not a helm repository 👍🏽 

`helm repo add cosmotech https://cosmo-tech.github.io/helm-charts`

File: README.md [hunk 2]
```diff
@@ -1,797 +1,29 @@
...
-
-You can define this root certificate either with CRT/PEM format or with a Java Key Store / JKS.
-
-To do so you need first to create a kubernetes secret with your certificate file, then you must to define the values in api.tlsTruststore.
-
-##### Example: Redis TLS with PEM custom root CA
-Create a kubernetes secret:
-``` bash
-kubectl -n mynamespace create secret generic redis-certificates-tls-secret --from-file=./rootCA.crt
+helm install mychartname cosmotech-api/chartname --version X.Y.Z
```
Comment (vysahjk) [inline README.md:25]: `helm install mychartname cosmotech/chartname --version X.Y.Z` instead

**Review by vysahjk — CHANGES_REQUESTED** [review]
there are some changes to do, but it is ok 👍🏽

File: README.md (hunk 1 above)
Comment (sellisd) [reply README.md]: ok, done, merci!

**Pull request merged**